mutagen/_riff.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017  Borewit
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+"""Resource Interchange File Format (RIFF)."""
+
+import struct
+from struct import pack
+
+from ._compat import text_type
+
+from mutagen._util import resize_bytes, delete_bytes, MutagenError
+
+
+class error(MutagenError):
+    pass
+
+
+class InvalidChunk(error):
+    pass
+
+
+def is_valid_chunk_id(id):
+    assert isinstance(id, text_type)
+
+    return ((len(id) <= 4) and (min(id) >= u' ') and
```

Comment:
The entire check is odd, why only the first character for the lower limit and the last char for the upper limit.

```` Python
if len(id) != 4:
       return False

    for i in range(0, 3):
        if id[i] < u' ' or id[i] > u'~':
            return False

    return True
````
✔